docker-images/jenkins/Dockerfile

```diff
@@ -46,7 +39,10 @@ usermod -aG sudo jenkins
 
 # Install microgateway toolkit
 COPY wso2am-micro-gw-toolkit-3.0.1.zip /wso2am-micro-gw-toolkit-3.0.1.zip
```

Comment:
@Aaquiff @KosalaHerath @rasanjaya85 @ThilinaManamgoda shall we add the Micro Gateway Toolkit's GA version to Bintray [1] and let this Dockerfile download this pack by default, when built? In order to allow the user to customize the build, let us provide the pack location URL as a Dockerfile build argument, so that when desired (e.g. when a pack with WSO2 Updates need to be packaged), the user can override the download location?

WDYT?

[1]: [WSO2 at Bintray](https://bintray.com/wso2/binaryGA)

Reply:
+1